### PR TITLE
Feature/temp

### PR DESCRIPTION
**Pull request by diggercam**
templating + new categorie

### CODE REVIEW

File: app/base/forms.py [hunk 1]
```diff
@@ -113,3 +113,74 @@ class RoadProDisplacementForm(FlaskForm):
...
+class InvestmentsForm(FlaskForm):
+    surface = IntegerField(
+        "Surface construite (m2)",
+        id="surface", validators=[DataRequired()])
+    activity = DecimalField(
+        "Type d'activité",
+        id="activity", validators=[DataRequired()])
+    consttype = DecimalField(
+        "Type de structure",
+        id="consttype", validators=[DataRequired()])
```
Comment (MartinHerr) [inline app/base/forms.py:183]: @diggercam I bet these fields are not Decimal fields. Maybe rather dropdowns or something similar?

File: app/base/forms.py [hunk 2]
```diff
@@ -113,3 +113,74 @@ class RoadProDisplacementForm(FlaskForm):
...
+    cars = DecimalField(
```
Comment (MartinHerr) [inline app/base/forms.py:184]: @diggercam I'd rather use an IntegerField for this one.

File: app/base/models.py [hunk 3]
```diff
@@ -132,3 +138,56 @@ class RoadProDisplacement(db.Model):
...
+
+    id = Column(Integer, primary_key=True)
+    surface = Column(Integer, nullable=False)
+    activity = Column(Numeric, nullable=False)
+    consttype = Column(Integer, nullable=False)
+    cars = Column(Integer, nullable=False)
+    user_id = Column(Integer, db.ForeignKey('User.id'), nullable=False)
+
+    def __repr__(self):
+        return f"InvestmentsValues('{self.paper}', '{self.carton}')"
```
Comment (MartinHerr) [inline app/base/models.py:193]: @diggercam update the `__repr__` method to return the correct table fields.
`return f"InvestmentValues('{self.surface}', '{self.activity}', <and so on>)"`

**Review by MartinHerr — CHANGES_REQUESTED** [review]

File: app/base/forms.py (hunk 1 above)
Comment (diggercam) [reply app/base/forms.py:183]: Yes, I guess dropdown is the best option, but need to work on it. We can keep it as it for now, I'm not planning to include Investments category for RSE Occitanie first POC, not needed for most of the companies.

File: app/base/forms.py (hunk 2 above)
Comment (diggercam) [reply app/base/forms.py:184]: Same here, should in the end be a dropdown with metal or béton

File: app/base/models.py [hunk 4]
```diff
@@ -132,3 +138,56 @@ class RoadProDisplacement(db.Model):
...
+        return f"ConsummablesConsumption('{self.paper}', '{self.carton}')"
+
+class InvestmentsValues(db.Model):
+
+    __tablename__ = 'InvestmentsValues'
+
+    id = Column(Integer, primary_key=True)
+    surface = Column(Integer, nullable=False)
+    activity = Column(Numeric, nullable=False)
+    consttype = Column(Integer, nullable=False)
```
[the remark this answers is not in the document]
Comment (diggercam) [reply app/base/models.py:188]: Yes, should be modified when we will have a clear view of the best way to implement the dropdown

File: app/base/models.py (hunk 3 above)
Comment (diggercam) [reply app/base/models.py:193]: done

File: app/base/templates/includes/sidebar.html [hunk 5]
```diff
@@ -44,6 +44,27 @@
...
+                </a>
+            </li>
+
+            <li class="nav-item nav-item {% if 'numerique' in segment %} active {% endif %}">
+                <a class="nav-link" href="/numerique.html">
+                    <i class="nc-icon nc-tablet-2"></i>
+                    <p>Numérique</p>
+                </a>
+            </li>
+
```
[the remark this answers is not in the document]
Comment (diggercam) [reply app/base/templates/includes/sidebar.html:67]: Line 54-59 ? Am I missing something ?

[the remark this answers is not in the document]
Comment (MartinHerr) [reply app/base/templates/includes/sidebar.html:67]: My bad

File: app/base/models.py (hunk 3 above)
Comment (MartinHerr) [reply app/base/models.py:193]: If it is done, then you need to push your modification to this branch.

**Comment by MartinHerr** [comment]
@diggercam j'ai réparé tous les éléments qui étaient cassés (je crois). La prochaine fois, si tu pouvais tester tes modifs avant de les pousser, pour vérifier qu'elles ne génèrent pas de régression, ce serait cool.

**Review by MartinHerr — APPROVED** [review]

**Pull request merged**